httpie/codec.py

```diff
@@ -0,0 +1,77 @@
+from argparse import Namespace
+from typing import Union
+
+from requests.utils import get_encoding_from_headers
+
+from .cli.constants import DEFAULT_FORMAT_BODY_CHARSET
+from .constants import UTF8
+from .context import Environment
+
+
+def detect(content: bytes) -> str:
+    """Detect the charset based on `content`.
+    Fallback to UTF-8 when no suitable encoding found.
+
+    """
+    from charset_normalizer import from_bytes
```

Comment:
Why making the import here? I am curious.

Reply:
At first it was to lazy import the module. But now that I thought about it, `request` 2.26.0+ will already import it. I(ll move it to the top, thank you for the catch.